### PR TITLE
Create authors' pages

### PR DESCRIPTION
**Pull request by robertkirsz**
Clicking on author's avatar redirects to a page with all of that author's posts

### CODE REVIEW

File: src/templates/authorPageTemplate.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    }),
+  }),
+}
+
+export default function AuthorPageTemplate({ pageContext, data, location }) {
+  const { author } = pageContext
+  const { title } = data.site.siteMetadata
+  const { posts } = data.allMarkdownRemark
+
+  // TODO: I'm fetching all posts and then filtering them by ID here
```
Comment (kzadurska) [inline src/templates/authorPageTemplate.js]: Let's improve this later on :)

File: src/components/Author.js [hunk 2]
```diff
@@ -8,7 +8,9 @@ import twitterLogo from '../../content/assets/twitter-logo.svg'
 
 const Author = ({ author, ...props }) => (
   <Div column itemsCenter listTop={16} {...props}>
-    <Avatar src={author.avatar.publicURL} alt={author.name} />
+    <Link to={`/authors/${author.id}`}>
+      <Avatar src={author.avatar.publicURL} alt={author.name} />
```
Comment (kzadurska) [inline src/components/Author.js]: I think I'd wrap the link around the `{author.name}` as well.

Comment (robertkirsz) [reply src/components/Author.js]: 👍

File: src/templates/authorPageTemplate.js (hunk 1 above)
Comment (robertkirsz) [reply src/templates/authorPageTemplate.js]: Improved already 💪

**Pull request merged**